src/metabase/server/routes/index.clj

```diff
@@ -25,18 +25,19 @@
   ;; https://stackoverflow.com/questions/14780858/escape-in-script-tag-contents/23983448#23983448
   (str/replace s #"(?i)</script" "</scr\\\\ipt"))
 
-(defn- fallback-localization [locale-or-name]
+(def ^:private fallback-localization
   (json/generate-string
    {"headers"
-    {"language"     (str locale-or-name)
+    {"language"     "en"
      "plural-forms" "nplurals=2; plural=(n != 1);"}
 
     "translations"
     {"" {"Metabase" {"msgid"  "Metabase"
                      "msgstr" ["Metabase"]}}}}))
 
 (defn- localization-json-file-name [locale-or-name]
-  (format "frontend_client/app/locales/%s.json" (str (i18n/locale locale-or-name))))
+  (format "frontend_client/app/locales/%s.json"
+          (str/replace (.toLanguageTag (i18n/locale locale-or-name)) \- \_)))
```

Comment:
Wouldn't hurt to leave a comment linking to this SO post, so anyone else reading this can get some context: https://stackoverflow.com/a/55965008